packages/blocks/src/components/import-page/import-page.ts

```diff
@@ -174,16 +173,19 @@ export async function importNotion(workspace: Workspace, file: File) {
             let id = 1;
             const titles: string[] = [];
             const rows: string[][] = [];
-            tableString?.split('\n').forEach((row, index) => {
+            tableString?.split(/\r\n|\r|\n/).forEach((row, index) => {
+              const rowArray = row.split(/,\s*(?![^"]*",)/);
```

Comment:
![image](https://github.com/toeverything/blocksuite/assets/59462000/d6cb853a-0e72-4311-8b11-6ba3c3c8aa8f)

Reply:
Separated by commas, but ignoring commas within quotes.

Reply:
```
",", ",,", ""
   ^     ^
```
What you need to match is two, but you only match one.

Reply:
Replaced with this regular   /,\s*(?=(?:(?:[^"]*"){2})*[^"]*$)/

Reply:
![image](https://github.com/toeverything/blocksuite/assets/59462000/ceb84878-8d10-4a1b-8678-602d3281113a)
It's never easy parsing a string. I suggest using libraries like [papaparse](https://github.com/mholt/PapaParse).